### DISCUSSION TITLE
The Big PHP Coding Style Discussion

### DISCUSSION THREAD

**Discussion by hackathi** [mistressofjellyfish/not-grocy#17] [Ideas]
Hi all,

after reading through https://www.php-fig.org/psr/psr-12/, I'm pretty much sold – even though I'm not too fond of how PSR-12 handles multiline expressions in `if`, `while` and friends. Also that else is on the same line as the closing `}` doesn't do separation very good.

So, maybe PSR-12 but with strousup if/else:

```php
if($x === null) {
    echo "foo";
}
else {
    echo "bar";
}
```

**Comment by mohe2015** [comment]
I would be fine with that.
